src/Core/tests/DeviceTests/Handlers/HandlerTestBase.Tests.cs

```diff
@@ -77,5 +78,18 @@ public async Task NullSemanticsClass()
 			var id = await GetValueAsync(view, handler => GetAutomationId(handler));
 			Assert.Equal(view.AutomationId, id);
 		}
+
+		[Fact(DisplayName = "Clip Initializes ContainerView Correctly")]
+		public async Task ContainerViewInitializesCorrectly()
+		{
+			var view = new TStub
+			{
+				ClipShape = new EllipseStub()
+			};
+
+			var handler = await CreateHandlerAsync(view);
+
+			Assert.NotNull(handler.ContainerView);
+		}
 	}
```

Comment:
Should we add more tests for the other shapes? Or is Ellipse sufficient?

Reply:
Is ok, because validate more the WrapperView creation than other stuff. Said that, never have enough tests, and could add more.

Reply:
makes sense, thanks!